homeassistant/components/emulated_hue/__init__.py

```diff
@@ -245,10 +245,11 @@ def is_entity_exposed(self, entity):
 
         return is_default_exposed or expose
 
-    def _load_json(filename):
-        """Wrapper, because we actually want to handle invalid json."""
-        try:
-            return load_json(filename)
-        except HomeAssistantError:
-            pass
-        return {}
+
+def _load_json(filename):
```

Comment:
linting suggested changing the method to a function, which I did now, but is this really the best approach?

Reply:
Looks good to me.